src/models/db.js

```diff
@@ -82,6 +96,23 @@ export default config => {
     if ('associate' in db[modelName]) {
       db[modelName].associate(db);
     }
+
+    const addDbHooks = () => {
+      if (!clientHooks) {
+        return;
+      }
+
+      if (clientHooks[modelName]) {
```

Comment:
As mentioned before, you can get the model name inside the hook itself. So you would simply do:

```javascript
Object.keys(db).forEach(modelName => {
  const model = db[modelName];
  if ('associate' in model) {
    model.associate(db);
  }

  if (!clientHooks) {
    return;
  }

  hooks.forEach(hook => {
    const clientHook = clientHooks[hook]
    clientHook && model.hook(hook, clientHook);
  }
});
```